src/pokete/classes/difficulty.py

```
@@ -53,20 +68,59 @@ def __init__(self):
+        
+        Returns:
+            float: Multiplier between 0.9 and 1.0; a higher average resonance produces greater relief (closer to 0.9).
+        """
         avg_res = sum(self.style_resonance.values()) / len(self.style_resonance)
         # Full resonance (1.0) gives 10% relief (0.9)
         relief = avg_res * 0.1
         return 1.0 - relief
 
-    def report_battle_detailed(self, rating, _turns, items_used, player_start_hp, player_end_hp, unique_moves, *, trainer_name=None, won=True):
-        """Processes high-fidelity battle telemetry for Style and Nemesis logic."""
+    def report_battle_detailed(self, rating, turns, items_used, player_start_hp, player_end_hp, unique_moves, *,
```

Ruff (0.14.14)

[warning] Unused method argument: `turns`

(ARG002)